aidial_adapter_bedrock/llm/errors.py

```diff
@@ -32,13 +33,9 @@ async def report_usage(self, choice: Choice) -> None:
                 stage.append_content(self.usage_message)
 
     def to_dial_exception(self) -> DialException:
-        return DialException(
-            status_code=422,
-            type="invalid_request_error",
+        return RequestValidationError(
             message=self.error_message,
             display_message=self.error_message,
-            code="invalid_argument",
```

Comment:
I still don't understand, if someone actually uses it, but inside `RequestValidationError` code will be empty, so we might keep this `code="invalid_argument"` for consistency

Reply:
fixed